src/screens/app/Components/UserCaption/UserCaption.js

```diff
@@ -1,23 +1,32 @@
 import React from 'react';
 import {Subheading} from 'react-native-paper';
 import {Text, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
```

Comment:
We shouldn't be needing the dependency I think. It's usually a syntax error that makes you download the dependency

Reply:
I'll look into this. Pretty sure we need prop-types to use PropTypes!